packages/nextjs/mocks/factory/index.ts

```diff
@@ -249,8 +246,12 @@ export class MockFactory {
   }
 }
 
+/** Converts { prop?: number } into { prop: number | undefined } */
```

Comment:
❓ Are these not equivalent as far as typescript is concerned?  Think I'm a bit lost as to what this affects.

Reply:
There are 2 differences with `Complete`,. First, the field is required, which means you get an error if you forget it. Second, you can specify `undefined` as a value, which normally isn't allowed for optional fields.

Technically, these differences could cause runtime problems, because it changes the behavior of `in` and `Object.keys`. But the convenience is nice ... I just wish there was a better way.